### PR TITLE
Output mock GPS from test script

### PR DESCRIPTION
**Pull request by emilytrau**
## Description

Modified the test data generator to output reasonable GPS locations.

## Screenshots

image

## Steps to Test

```
cd Raspi/Test
python mqtt_test.py -t 10
```

### CODE REVIEW

File: Raspi/Test/das_data_generator.py [hunk 1]
```diff
@@ -9,14 +9,22 @@ def send_fake_data(send_data_func, duration, rate, immitate_teensy=False):
...
     while True:
         current_time = round(time.time(), 2)
         total_time = round(current_time - start_time, 2)
 
         # Generate data message
-        data = "gps=1&gps_lat=00&gps_long=00&gps_alt=00&gps_course=00&"
+        data = "gps=1"
+        data += "&gps_lat=" + str(gps_lat)
+        data += "&gps_long=" + str(gps_long)
+        gps_lat += random.randint(0, 20) * 0.00001
```
Comment (khanguslee) [inline Raspi/Test/das_data_generator.py]: I would leave a comment stating why you chose those values. (eg. Random distance up to x metres away etc.)

**Review by khanguslee — CHANGES_REQUESTED** [review]
Good idea! This will probably not cause the dashboard to freak out if we run this script now! Just a small change!

File: Raspi/Test/das_data_generator.py (hunk 1 above)
Comment (emilytrau) [reply Raspi/Test/das_data_generator.py]: I added some comments and changed the code to do movement based on metres instead of an arbitrary degree.

**Comment by khanguslee** [comment]
Definitely works as intended! Very cool! 

Screen Shot 2020-01-12 at 3 35 40 pm

**Review by khanguslee — APPROVED** [review]
Looks good to me! Feel free to create a card on software board with what you did here!

**Pull request merged**